guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -144,12 +144,27 @@ endif::[]
 ----
 # leapp preupgrade
 ----
-The first run is expected to fail but report issues.
+If `leapp preupgrade` inhibits the upgrade with an unsupported network configuration because there are multiple legacy named network interfaces, follow the instructions shown by Leapp to rename the interfaces, followed by an installer run to reconfigure {Project} to use the new interface names:
```

Comment:
"Unsupported network configuration" is the output Leapp gives you -- is there a way to highlight it here as such?
In my original suggestion I quoted it, but not sure this is the correct way to do it?

Reply:
@evgeni I can make it bold.